### PR TITLE
CMC fixed prereqs and synopsis

### PR DESCRIPTION
**Pull request by CitMC**
### Change scope

The changes in this Pull Request include:

- [x] Small language/grammar fixes
- [ ] Small content fixes 
- [ ] Addition of new content
- [ ] Sample code/command updates
- [ ] Platform fixes
- [ ] Other

### CODE REVIEW

**Comment by CitMC** [comment]
@coldice  merge asap

File: academy/ibc/clients.md [hunk 1]
```diff
@@ -5,7 +5,17 @@ description:
 tag: deep-dive
 ---
 
-## Clients
+# Clients
+
+<HighlightBox type="synopsis">
```
Comment (xavierlepretre) [inline academy/ibc/clients.md:10]:
```suggestion
<HighlightBox type="learning">
```

File: academy/ibc/channels.md [hunk 2]
```diff
@@ -9,10 +9,10 @@ tag: deep-dive
 
 <HighlightBox type="synopsis">
```
Comment (xavierlepretre) [inline academy/ibc/channels.md:10]:
```suggestion
<HighlightBox type="learning">
```

File: academy/ibc/gorelayer.md [hunk 3]
```diff
@@ -7,16 +7,26 @@ tag: deep-dive
...
+
+* Install Go.
+* Install Docker.
+* Install Rust.
+
+For all installations, please see the [setup page](../course-ida/setup.md).
+
+</HighlightBox>
+
 <HighlightBox type="synopsis">
```
Comment (xavierlepretre) [inline academy/ibc/gorelayer.md:22]:
```suggestion
<HighlightBox type="learning">
```

File: academy/ibc/hermesrelayer.md [hunk 4]
```diff
@@ -7,6 +7,27 @@ tag: deep-dive
...
+
+* Install Go.
+* Install Docker.
+* Install Rust.
+
+For all installations, please see the [setup page](../course-ida/setup.md).
+
+</HighlightBox>
+
+<HighlightBox type="synopsis">
```
Comment (xavierlepretre) [inline academy/ibc/hermesrelayer.md:22]:
```suggestion
<HighlightBox type="learning">
```

File: academy/ibc/ibc-tao-dev.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ tag: deep-dive
 
 <HighlightBox type="synopsis">
```
Comment (xavierlepretre) [inline academy/ibc/ibc-tao-dev.md:10]:
```suggestion
<HighlightBox type="learning">
```

File: academy/ibc/ibc-tooling.md [hunk 6]
```diff
@@ -9,14 +9,12 @@ tag: deep-dive
 
 <HighlightBox type="synopsis">
```
Comment (xavierlepretre) [inline academy/ibc/ibc-tooling.md:10]:
```suggestion
<HighlightBox type="learning">
```

File: academy/ibc/ica.md [hunk 7]
```diff
@@ -1,19 +1,21 @@
+---
+title: "Interchain Accounts"
+order: 
+description: 
+tag: deep-dive
+---
+
 # Interchain Accounts
 
 <HighlightBox type="synopsis">
```
Comment (xavierlepretre) [inline academy/ibc/ica.md:10]:
```suggestion
<HighlightBox type="learning">
```

File: academy/ibc/relayerintro.md [hunk 8]
```diff
@@ -7,6 +7,12 @@ tag: deep-dive
 
 # Relaying in General
 
+<HighlightBox type="synopsis">
```
Comment (xavierlepretre) [inline academy/ibc/relayerintro.md:10]:
```suggestion
<HighlightBox type="learning">
```

File: academy/ibc/token-transfer.md [hunk 9]
```diff
@@ -9,15 +9,10 @@ tag: deep-dive
 
 <HighlightBox type="synopsis">
```
Comment (xavierlepretre) [inline academy/ibc/token-transfer.md:10]:
```suggestion
<HighlightBox type="learning">
```

File: academy/ibc/what-is-ibc.md [hunk 10]
```diff
@@ -9,15 +9,13 @@ tag: fast-track
 
 <HighlightBox type="synopsis">
```
Comment (xavierlepretre) [inline academy/ibc/what-is-ibc.md:10]:
```suggestion
<HighlightBox type="learning">
```

**Pull request merged**